### ISSUE TITLE
Suggestion: Internet Archive

### ISSUE THREAD

**Issue by catalandres** [LK64076007A/driftwood#1]
Add a workflow to allow for serving podcasts from the Internet Archive (unlimited and free, unlike GitHub).

**Comment by LK64076007A** [comment]
Thanks for the suggestion. 

To clarify, you mean have the podcast files hosted on IA and keep the website and RSS feed on GitHub?

**Comment by catalandres** [comment]
Exactly.

**Comment by LK64076007A** [comment]
Good idea. I'll look into the best way to include that change in the project and update it. 

In the mean time, if you wanted to do it now, I think all you'd need to do is put the full file URL at IA in the YAML front matter for the episode posts and change `<enclosure url="{{ site.url }}{{ site.baseurl }}{{ post.file }}" length="{{ post.length }}" type="audio/x-m4a"/>` to `<enclosure url="{{ post.file }}" length="{{ post.length }}" type="audio/x-m4a"/>` in podcast.rss.
